packages/jest-core/src/TestScheduler.ts

```diff
@@ -193,7 +193,8 @@ export default class TestScheduler {
     contexts.forEach(context => {
       const {config} = context;
       if (!testRunners[config.runner]) {
-        const Runner: typeof TestRunner = require(config.runner);
+        const Runner: typeof TestRunner =
+          require(config.runner).default || require(config.runner);
```

Comment:
we have a helper for this: https://github.com/facebook/jest/blob/dbb12903801f7423f0273d4c1872294e9dbbc5ce/packages/jest-util/src/interopRequireDefault.ts

Reply:
Ah ok cool. Later on I'll try to make use of this and add a changelog entry.